packages/fuel-indexer-plugin/src/wasm.rs

```diff
@@ -75,15 +75,15 @@ pub trait Entity<'a>: Sized + PartialEq + Eq + std::fmt::Debug {
         }
     }
 
-    fn load(id: u64) -> Option<Self> {
+    fn load(id: UID) -> Option<Self> {
         unsafe {
-            let buf = id.to_le_bytes();
-            let mut buflen = (buf.len() as u32).to_le_bytes();
+            let buff = bincode::serialize(&id.to_string()).unwrap();
```

Comment:
I _think_ we can remove this `.to_string()` call here as `SizedAsciiString` provides an `impl` for `Serialize`: https://github.com/FuelLabs/fuels-rs/blob/df21541b679bb90a1ee3e54f34a67b7037b40be7/packages/fuels-core/src/types/core/sized_ascii_string.rs#L162-L169.

Additionally, can we replace the `.unwrap()` with an `.expect()`?